app/components/Icons/MaterialIcons.tsx

```diff
@@ -0,0 +1,8 @@
+import { IconBaseProps } from "react-icons";
+import { MdAdd } from "react-icons/md"
+
+export const MdIconAdd = (props: IconBaseProps) => {
```

Comment:
nits: ここはシンプルに `MdIconAdd` -> `AddIcon` で良さそうです
```suggestion
export const AddIcon = (props: IconBaseProps) => {
```

Reply:
他のコンポーネントと被ったりは大丈夫そうですか？

Reply:
修正しました
https://github.com/Greek-Academy/alpha-dojo-v2/pull/3/commits/97f50201087c50483bc469e7f92a52453bb913a4